hypothesis/via
# Add upstream cache control conversion like we have in ViaHTML

For: https://github.com/hypothesis/checkmate/issues/216

This is mostly to allow us to emit "private" to prevent Cloudflare from caching this to prevent cache priming attacks, but it also means that browsers will be able to cache things.

## Testing notes

 * Run `make dev`
 * Visit: http://localhost:9080/http://example.com
 * You should see `Cache-Control: private, max-age=604800` come back
 * Visit http://localhost:9080/https://www.google.com/?gws_rd=ssl
 * You should see absolute gibberish and `Cache-Control: private` come back (or whatever Google decides to send)
